src/plugins/cosmos/components/modals/GetStarted/views/LearnMore.tsx

```diff
@@ -0,0 +1,151 @@
+import { ArrowBackIcon } from '@chakra-ui/icons'
+import { Box, Flex } from '@chakra-ui/layout'
+import { Button, IconButton } from '@chakra-ui/react'
+import { useSteps } from 'chakra-ui-steps'
+import { DefiModalHeader } from 'plugins/cosmos/components/DefiModalHeader/DefiModalHeader'
+import { useHistory } from 'react-router-dom'
+import rewards from 'assets/rewards.svg'
+import risk from 'assets/risk.svg'
+import withdraw from 'assets/withdraw.svg'
+import { CarouselDots } from 'components/CarouselDots/CarouselDots'
+import { SlideTransition } from 'components/SlideTransition'
+import { Text } from 'components/Text'
+import { useModal } from 'context/ModalProvider/ModalProvider'
+
+const STEPS_LENGTH = 3
+const STEP_TO_ELEMENTS_MAPPING = {
+  1: {
+    bodies: [
+      'defi.modals.learnMore.bodies.rateFluctuationInfo',
+      'defi.modals.learnMore.bodies.amountStakingInfo',
+      'defi.modals.learnMore.bodies.withdrawInfo'
+    ],
+    header: 'defi.modals.learnMore.headers.aboutStakingRewards',
+    headerImageSrc: rewards
+  },
+  2: {
+    bodies: ['defi.modals.learnMore.bodies.unbondingInfo'],
+    header: 'defi.modals.learnMore.headers.unstaking',
+    headerImageSrc: withdraw
+  },
+  3: {
+    bodies: [
+      'defi.modals.learnMore.bodies.slashingInfo',
+      'defi.modals.learnMore.bodies.partnerInfo'
+    ],
+    header: 'defi.modals.learnMore.headers.risks',
+    headerImageSrc: risk
+  }
+}
```

Comment:
Using an object to represent this feels strange to me.

If we use an array we can get rid of the strangeness with `let currentStep = activeStep as 1 | 2 | 3` etc.

For e.g.:

```ts
const STEP_TO_ELEMENTS_MAPPING = [
  {
    bodies: [
      'defi.modals.learnMore.bodies.rateFluctuationInfo',
      'defi.modals.learnMore.bodies.amountStakingInfo',
      'defi.modals.learnMore.bodies.withdrawInfo'
    ],
    header: 'defi.modals.learnMore.headers.aboutStakingRewards',
    headerImageSrc: rewards
  },
  {
    bodies: ['defi.modals.learnMore.bodies.unbondingInfo'],
    header: 'defi.modals.learnMore.headers.unstaking',
    headerImageSrc: withdraw
  },
  {
    bodies: [
      'defi.modals.learnMore.bodies.slashingInfo',
      'defi.modals.learnMore.bodies.partnerInfo'
    ],
    header: 'defi.modals.learnMore.headers.risks',
    headerImageSrc: risk
  }
]
```

Then (see below)

Reply:
Yeah, the rationale to not using a tuple was for readability, not to have these "big" objects and having to have the correct order, explicitly setting the indexor key seemed nicer for that. But that came at the cost of having to fight with TSC, so... switched to a tuple!